packages/nimble-components/src/table-column/text/index.ts

```diff
@@ -0,0 +1,54 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { StringField } from '../../table/types';
+import { TableColumn } from '../base';
+import { styles } from '../base/styles';
+import { template } from '../base/template';
+import { cellStyles } from './styles';
+import { cellTemplate } from './template';
+
+export type TableColumnTextCellData = StringField<'value'>;
```

Comment:
Calling this type "CellData" is a bit ambiguous since there is so many pieces of data that go into a cell and that come from different places. Do you think calling this "CellRecord" would make sense, i.e. "TableColumnTextCellRecord"?

Reply:
Updated.